src/txn/txn.c

```diff
@@ -881,23 +881,33 @@ __txn_commit_timestamps_usage_check(WT_SESSION_IMPL *session, WT_TXN_OP *op, WT_
           "no timestamp provided for an update to a "
           "table configured to always use timestamps once they are first used"));
 
-    if (FLD_ISSET(ts_flags, WT_DHANDLE_TS_ORDERED) && txn_has_ts && prev_op_durable_ts > op_ts)
+    /*
```

Comment:
How about adding a similar assert condition to all the other cases that exist instead of only to the `ORDERED`?

Reply:
Good idea, thanks for the suggestion.